src/main/java/org/cbioportal/proxy/ProxyController.java

```diff
@@ -226,54 +213,30 @@ private <T> ResponseEntity<T> exchangeData(String body, URI uri, HttpMethod meth
         return restTemplate.exchange(uri, method, new HttpEntity<>(body, httpHeaders), responseType);
     }
 
+    private boolean isKnownHost(String pathInfo) {
+        String[] knownHosts = {
+            this.genomeNexusUrl,
+            this.genomeNexusGrch38Url,
+            "http://mutationaligner.org/api",
+            "http://docs.cbioportal.org/news"
```

Comment:
should we fix cors on the docs site?

Reply:
Looks like Github pages now allows CORS! See cBioPortal/cbioportal-frontend#4941